rules/python-click.mk

```diff
@@ -7,11 +7,10 @@
 # TODO: If we upgrade to a distro which provides a version >= 6.7 we will no
 # longer need to build this.
```

Comment:
Seems no need to build

Reply:
@qiluo-msft: I addressed this in a separate comment. You can delete or resolve this comment, as it is a duplicate.